dot/sync/chain_processor.go

```diff
@@ -95,99 +95,109 @@ func (s *chainProcessor) processReadyBlocks() {
 // processBlockData processes the BlockData from a BlockResponse and
 // returns the index of the last BlockData it handled on success,
 // or the index of the block data that errored on failure.
-func (s *chainProcessor) processBlockData(bd *types.BlockData) error {
-	if bd == nil {
-		return ErrNilBlockData
-	}
+func (c *chainProcessor) processBlockData(blockData types.BlockData) error { //nolint:revive
```

Comment:
curious why you renamed `s` to `c`?

👍🏼 for renaming `bd` to `blockData`

Reply:
> curious why you renamed `s` to `c`?

Copy-pastaing my answer above:
for method receivers, I made peace with having short or single letter variable names.
Also see https://github.com/golang/go/wiki/CodeReviewComments#receiver-names :wink: (also just learnt that tbh)

Reply:
why did we change from `*types.BlockData` to `types.BlockData`?